### ISSUE TITLE
version with wildcards and determinism - AssemblyInfo.cs

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#39354]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/idea/777498/version-with-wildcards-and-determinism-assemblyinf.html)._

---
Hello, 

I have a sugestion:
update the message 
// You can specify all the values or you can default the Build and Revision Numbers
// by using the '*' as shown below:
// [assembly: AssemblyVersion(&quot;1.0.*&quot;)]
// **and set the &lt;Determinism&gt;false&lt;/Determinism&gt; in project file as shown below:**
&lt;PropertyGroup&gt;
    &lt;Configuration Condition=&quot; '$(Configuration)' == '' &quot;&gt;Debug&lt;/Configuration&gt;
    &lt;Platform Condition=&quot; '$(Platform)' == '' &quot;&gt;AnyCPU&lt;/Platform&gt;
    &lt;ProjectGuid&gt;{559EFE43-8D23-4FEE-A9E6-A2D70AB5C855}&lt;/ProjectGuid&gt;
    &lt;OutputType&gt;Library&lt;/OutputType&gt;
    &lt;AppDesignerFolder&gt;Properties&lt;/AppDesignerFolder&gt;
    &lt;RootNamespace&gt;DataAccess&lt;/RootNamespace&gt;
    &lt;AssemblyName&gt;DataAccess&lt;/AssemblyName&gt;
    &lt;TargetFrameworkVersion&gt;v4.7.2&lt;/TargetFrameworkVersion&gt;
    &lt;FileAlignment&gt;512&lt;/FileAlignment&gt;
//    **&lt;Deterministic&gt;false&lt;/Deterministic&gt;**
  &lt;/PropertyGroup&gt;

This will avoid the error message:
Error	CS8357	The specified version string contains wildcards, which are not compatible with determinism.
 Either remove wildcards from the version string, or disable determinism for this compilation	
DataAccess	C:\Users\anton\source\repos\OperatorNotificationSystem\DataAccess\Properties\AssemblyInfo.cs	35	Active

thanks,

Antonio Herrero Lopes

---
#### Jane Wu [MSFT] on 10/15/2019, 07:27 PM: 

<p>Thank you for taking the time to provide your suggestion. We will do some preliminary checks to make sure we can proceed further.  We’ll provide an update once the issue has been triaged by the product team.</p>

**Comment by jinujoseph** [comment]
cc @jmarolf  do you know where the template for this file is ? AssemblyInfo template

**Comment by tmat** [comment]
I don't think we should make this part of a template. We want customers to use deterministic build and stop using `AssemblyVersion(`*`)`, not the other way around.

**Comment by CyrusNajmabadi** [comment]
CLosing out due to lack of feedback.
